src/main/features/response/views/confirmation.njk

```diff
@@ -9,40 +10,41 @@
+        <p class="receipt-download-container">
+          <a href="{{ ResponsePaths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download your response') }}</a> (PDF)
         </p>
       </div>
 
       <h2 class="bold-large form-group">{{ t('Your response') }}</h2>
 
-      <p>{{ t('We’ve emailed {{ claimant }} your defence, explaining why you reject the claim.', { claimant: claim.claimData.claimant.name }) | safe}}</p>
-
-      <p>{{ t('View the <a href="{{ link }}">claim against you and your response.</a>', { link: '/dashboard' }) | safe }}</p>
+      {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
+        {% include "./confirmation/full-admission.njk" %}
+      {% endif %}
+      {% if claim.response.responseType === domain.ResponseType.FULL_DEFENCE %}
+        {% include "./confirmation/full-defence.njk" %}
+      {% endif %}
 
       <h2 class="heading-medium">{{ t('What happens next') }}</h2>
-      <p>{{ t('If {{ claimant }} accepts your defence the claim will be cancelled.', { claimant: claim.claimData.claimant.name }) | safe }}</p>
-      <p>{{ t('If they reject your defence the case will be reviewed and might go to a hearing.') }}</p>
-
-      {% if featureToggles.offer %}
-        <h2 class="heading-medium">{{ t('Settle out of court') }}</h2>
-        <p>{{ t('You may still <a href="{{ link }}">settle the claim out of court</a>. For example you could offer to fix goods you sold the claimant or suggest a payment.',
-            { link: OfferPaths.settleOutOfCourtPage.evaluateUri({ externalId: claim.externalId }) }) | safe }}</p>
-        <p>{{ t('You can avoid getting a County Court Judgment if the claimant accepts your offer.') }}</p>
-      {% endif %}
 
-      {% if claim.response.freeMediation === YesNoOption.YES.option %}
-        <h2 class="heading-medium">{{ t('Mediation') }}</h2>
-        <p>{{ t('You have chosen to use the free mediation service to resolve the dispute with {{ claimant }}. We’ll ask them if they want to mediate.', { claimant: claim.claimData.claimant.name  }) }}</p>
-        <p>{{ t('If they agree to mediation we’ll email you to arrange an appointment.') }}</p>
-        <p>{{ t('If you can reach an agreement with {{ claimant }} you won’t need to go to a hearing.', { claimant: claim.claimData.claimant.name  }) }}</p>
+      {% if claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
+        {% include "./confirmation/full-admission-steps.njk" %}
+      {% endif %}
+      {% if claim.response.responseType === domain.ResponseType.FULL_DEFENCE %}
+        {% include "./confirmation/full-defence-steps.njk" %}
       {% endif %}
-      {{ supportWidget() }}
 
       {% include "includes/service_survey_link.njk" %}
+
+      <p><a class="button" href="{{ DashboardPaths.dashboardPage.uri }}">{{ t('Go to your account') }}</a></p>
```

Comment:
I don't think this should be in a p tag as it's not really text/content. can you try just using form-group to see if you get the same effect?

Reply:
We could use `from-group` but that is not ideal solution as there is not form element either.

Technically `<a>` is a text element in HTML despite being styled out as button by the CSS :)